core-bundle/tests/Functional/app/config/config_test.yml

```diff
@@ -10,6 +10,7 @@ framework:
     translator: { fallbacks: ['%locale%'] }
     secret: '%env(APP_SECRET)%'
     router:
+        utf8: true
```

Comment:
Can you explain this change? This enabled `utf8` for all routes (Symfony-wide). The Contao routes already have uft8 enabled through our `RouteProvider`.

Reply:
Not setting the option is deprecated:

> Deprecated not setting the "framework.router.utf8" configuration option as it will default to true in Symfony 6.0

https://github.com/symfony/symfony/blob/5.2/UPGRADE-5.1.md#frameworkbundle